WeakAuras/BuffTrigger2.lua

```diff
@@ -1198,8 +1198,8 @@ end
 local function markForTriggerInfo(triggerInfo, unit)
   if triggerInfo.fetchRaidMark then
     local rt = GetRaidTargetIndex(unit)
-    if rt then
-      return "{rt" .. GetRaidTargetIndex(unit) .. "}"
+    if rt and rt > 1 and rt < 9 then
```

Comment:
i think you should do this, otherwise skull marks will mysteriously vanish
```suggestion
    if rt and rt >= 1 and rt < 9 then
```

Reply:
either that or im misremembering how raid target indices work